BAM/src/main/java/com/revature/bam/controller/CurriculumController.java

```diff
@@ -186,6 +187,22 @@ public CurriculumController(CurriculumService cs, CurriculumSubtopicService css,
 		curriculum.setCurriculumVersion(c.getMeta().getCurriculum().getCurriculumVersion());
 		curriculum.setIsMaster(c.getMeta().getCurriculum().getIsMaster());
 
+		if(curriculum.getIsMaster() == 1) {
+			List<Curriculum> curriculumList = curriculumService.findAllCurriculumByName(curriculum.getCurriculumName());
+			Curriculum prevMaster = null;
+
+			for (int i = 0; i < curriculumList.size(); i++) {
+				if (curriculumList.get(i).getIsMaster() == 1)
+						prevMaster = curriculumList.get(i);
+			}
+			if (prevMaster != null) {
+				prevMaster.setIsMaster(0);
+				curriculumService.save(prevMaster);
+			} else {
+					LogManager.getRootLogger().error(prevMaster);
```

Comment:
Should there be an explicit logging call here? Check with the logging guys (Allan and David) to see if this belongs here.

Reply:
removed the logging call

Reply:
I think explicit log calls are fine, but be sure to include both the class and method names so it can be easily searched through Splunk.